ansible/roles_ocp_workloads/ocp4_workload_tracing_jaeger/defaults/main.yml

```diff
@@ -30,10 +30,10 @@ ocp4_workload_tracing_jaeger_namespace: ""
 
 # Catalog Source Name when using a catalog snapshot. This should be unique
 # in the cluster to avoid clashes
-ocp4_workload_tracing_jaeger_catalogsource_name: community-operators-snapshot-tracing_jaeger
+ocp4_workload_tracing_jaeger_catalogsource_name: redhat-operators-snapshot-tracing_jaeger
```

Comment:
this needs to be changed to the community.

Reply:
Done in subsequent PR